UI/window-basic-settings.cpp

```diff
@@ -921,6 +921,13 @@ OBSBasicSettings::OBSBasicSettings(QWidget *parent)
 
 	UpdateAudioWarnings();
 	UpdateAdvNetworkGroup();
+
+	/* Reduce the height of the window if too tall compared to the screen
+	 * size (e.g. 720p) with potential desktop decoration (e.g. taskbar) */
+	int max_height = round(
+		QGuiApplication::primaryScreen()->size().height() * 95 / 100);
```

Comment:
Does primaryScreen in this case get the primary display or the display that OBS is actually running on?

Reply:
The primary display.

Reply:
@tytan652 We should base this off the dimensions of the screen that the settings dialog is actually opening on

Reply:
Made the change but it needs to be tested on Windows.